cmd/juju/cloud/list.go

```diff
@@ -38,23 +38,40 @@ type listCloudsCommand struct {
 // listCloudsDoc is multi-line since we need to use ` to denote
 // commands for ease in markdown.
 var listCloudsDoc = "" +
-	"Output includes fundamental properties for each cloud known to the\n" +
-	"current Juju client: name, number of regions, default region, type,\n" +
-	"and description.\n" +
-	"\nThe default behaviour is to show clouds available on the current controller,\n" +
-	"or another controller specified using --controller.\n" +
-	"\nIf --local is specified, the output shows public clouds known to Juju out of the box;\n" +
-	"these can be used to bootstrap a controller. Clouds may change between Juju versions.\n" +
-	"In addition to these public clouds, the 'localhost' cloud (local LXD) is also listed.\n" +
-	"If you supply a controller name the clouds known on the controller will be displayed.\n" +
-	"\nThis command's default output format is 'tabular'.\n" +
-	"\nCloud metadata sometimes changes, e.g. AWS adds a new region. Use the\n" +
-	"`update-clouds` command to update the current Juju client accordingly.\n" +
-	"\nUse the `add-cloud` command to add a private cloud to the list of\n" +
+	"Display the clouds known to the current Juju client, as well as the number of region\n" +
+	"that each cloud supports, their default regions, their types, and short descriptions.\n" +
```

Comment:
I preferred the original opening sentence to be honest. More concise and meaningful in that it mentioned what was displayed - fundamental cloud properties.

Reply:
Have reverted, but expanded the line-width to what Chris had been using.